i18n/validate.py

```diff
@@ -175,24 +185,18 @@ def report_problems(filename, problems):
     `problems` is a list of tuples as returned by `check_messages`.
 
     """
-    if problems:
-        problem_file = filename.replace(".po", ".prob")
-        id_filler = textwrap.TextWrapper(width=79, initial_indent="  msgid: ", subsequent_indent=" " * 9)
-        tx_filler = textwrap.TextWrapper(width=79, initial_indent="  -----> ", subsequent_indent=" " * 9)
-        with codecs.open(problem_file, "w", encoding="utf8") as prob_file:
-            for problem in problems:
-                desc, msgid = problem[:2]
-                prob_file.write(u"{}\n{}\n".format(desc, id_filler.fill(msgid)))
-                for translation in problem[2:]:
-                    prob_file.write(u"{}\n".format(tx_filler.fill(translation)))
-                prob_file.write(u"\n")
-
-        log.error(" %s problems in %s, details in .prob file", len(problems), filename)
-    else:
-        dup_filename = filename.replace('.po', '.dup')
-        if os.path.exists(dup_filename):
-            log.warning(" Duplicates found in %s, details in .dup file", dup_filename)
-        log.info(" No problems found in %s", filename)
+    problem_file = filename.replace(".po", ".prob")
+    id_filler = textwrap.TextWrapper(width=79, initial_indent="  msgid: ", subsequent_indent=" " * 9)
+    tx_filler = textwrap.TextWrapper(width=79, initial_indent="  -----> ", subsequent_indent=" " * 9)
+    with codecs.open(problem_file, "w", encoding="utf8") as prob_file:
+        for problem in problems:
+            desc, msgid = problem[:2]
+            prob_file.write(u"{}\n{}\n".format(desc, id_filler.fill(msgid)))
+            for translation in problem[2:]:
+                prob_file.write(u"{}\n".format(tx_filler.fill(translation)))
+            prob_file.write(u"\n")
+
+    log.error(" %s problems in %s, details in .prob file", len(problems), filename)
```

Comment:
1. Remove the preceding space.
2. Pick a quote style—single or double—and stick with it.

Reply:
1.) Most of the log messages in this file have that preceding space for some reason. I'm not sure why it's there, so I'd prefer not to change it.

2.) The quote styles are mixed throughout the file. It's out of scope for me to update them all to be consistent.